backend/src/agents/__init__.py

```
@@ -56,30 +56,20 @@ def memory_to_xml(memory):
     )
 
 
-def graph_builder(
+def init_graph(
     tools: list[BaseTool] = [],
```

Ruff (0.14.13)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)